src/main/java/com/uber/jenkins/phabricator/PhabricatorNotifier.java

```diff
@@ -191,6 +198,9 @@ public final boolean perform(final AbstractBuild<?, ?> build, final Launcher lau
         // Read coverage data to send to Harbormaster
         resultProcessor.processCoverage(coverageProvider, diff.getChangedFiles());
 
+        // Read lint results to send to Harbormaster
+        resultProcessor.processLintResults(lintFile, lintFileSize);
```

Comment:
Looks like we don't respect `this.processLint`. Probably should just guard this with `if (this.processLint) {...`